ChatQnA/kubernetes/intel/cpu/xeon/manifest/chatqna-guardrails.yaml

```diff
@@ -1252,22 +1252,14 @@ spec:
           env:
             - name: LLM_SERVER_HOST_IP
               value: chatqna-tgi
-            - name: LLM_SERVER_PORT
-              value: "2080"
             - name: RERANK_SERVER_HOST_IP
               value: chatqna-teirerank
-            - name: RERANK_SERVER_PORT
-              value: "2082"
             - name: RETRIEVER_SERVICE_HOST_IP
               value: chatqna-retriever-usvc
             - name: EMBEDDING_SERVER_HOST_IP
               value: chatqna-tei
-            - name: EMBEDDING_SERVER_PORT
-              value: "2081"
             - name: GUARDRAIL_SERVICE_HOST_IP
               value: chatqna-guardrails-usvc
-            - name: GUARDRAIL_SERVICE_PORT
-              value: "9090"
```

Comment:
we still needs GAURDRAILS_SERVICE_PORT as env variable, because K8S service chatqna-guardrails-usvc's port number is 9090, different than the default 80